lib/portlayer/exec/container.go

```diff
@@ -258,6 +272,11 @@ func (c *Container) Commit(ctx context.Context, sess *session.Session, h *Handle
 
 		c.State = *h.State
 		commitEvent = events.ContainerStarted
+
+		// refresh the struct with what propery collector provides
+		if err := c.Refresh(); err != nil {
```

Comment:
True out of band operations will cause failure. That's why we will introduce retry logic to the personality.